gazebo_ros2_control_demos/urdf/test_cart_position.xacro.urdf

```diff
@@ -48,6 +48,34 @@
     <limit effort="1000.0" lower="-15" upper="15" velocity="30"/>
     <dynamics damping="0.0" friction="0.0"/>
   </joint>
+  <link name="imu">
+    <inertial>
+      <mass value="0.1"/>
+      <inertia ixx="1.0" ixy="0" ixz="0" iyy="1.0" iyz="0" izz="1.0"/>
+    </inertial>
+    <!-- <visual>
+      <origin xyz="0 0 0"/>
+      <geometry>
+        <box size="0.1 0.1 0.1"/>
+      </geometry>
+      <material name="red">
+        <color rgba="0.8 0 0 1"/>
+      </material>
+    </visual> -->
```

Comment:
Can we remove this ? or add a comment why this is usefull ?

Reply:
Sorry I'll remove the commented lines, they are not needed.

Reply:
Also the whole imu for the position cart, I didn't understand